amf-aml/shared/src/main/scala/amf/aml/internal/render/plugin/AMLDialectRenderingPlugin.scala

```diff
@@ -48,4 +59,9 @@ class AMLDialectRenderingPlugin extends AMFRenderPlugin {
   override def defaultSyntax(): String = `application/yaml`
 
   override def mediaTypes: Seq[String] = Seq(`application/yaml`)
+
+  override protected def unparseAsYDocument(unit: BaseUnit,
```

Comment:
This is weird, what is this case?

Reply:
Just be complaint with the interface to avoid duplicate code. Is unreachable as the emit method is overrided.

Reply:
hmmm we should probably change this (Out of Scope)